Apps/Sandcastle/gallery/Side by Side Camera.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Use Viewer to start building new applications or easily embed Cesium into existing applications.">
+    <meta name="cesium-sandcastle-labels" content="Beginner, Showcases">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+    #Container {
+          display: flex;
+          width: 100%;
+          height: 100%;
+      }
+      #View3D {
+          display: inline-block;
+          width: 100%;
+      }
+      #View2D {
+          display: inline-block;
+          width: 100%;
+      }
+</style>
+<div id="Container" class="fullSize">
+    <div id="View3D"></div>
```

Comment:
Our coding style for HTML id's is Camel Case, not Pascal Case. 

I don't think this or any HTML standards are mentioned in our Coding Guide. It would probably worth noting that we Camel Case HTML ids, and hyphenate CSS classes. Additionally, avoid inline `style` attributes, and prefer styling the class or id instead.